components/ca-list.css

```diff
@@ -0,0 +1,199 @@
+    color: #333;
+    white-space: nowrap;
+    text-decoration: none;
+}
+
+ca-list .button-group .dropdown-menu > li:hover {
+    background-color: #e6e6e6;
+}
+
+ca-list .ca-list-filter {
+    display: none;
+    background: url(data:image/svg+xml;base64,PD94bWwgdmVyc2lvbj0iMS4wIiA/PjxzdmcgaGVpZ2h0PSIxNnB4IiB2ZXJzaW9uPSIxLjEiIHZpZXdCb3g9IjAgMCAyNCAyNCIgd2lkdGg9IjE2cHgiIHhtbG5zPSJodHRwOi8vd3d3LnczLm9yZy8yMDAwL3N2ZyIgeG1sbnM6eGxpbms9Imh0dHA6Ly93d3cudzMub3JnLzE5OTkveGxpbmsiPjxnIGZpbGw9Im5vbmUiIGZpbGwtcnVsZT0iZXZlbm9kZCIgc3Ryb2tlPSJub25lIiBzdHJva2Utd2lkdGg9IjEiPiAgICA8ZyB0cmFuc2Zvcm09InRyYW5zbGF0ZSgtNTM5LjAwMDAwMCwgLTQwNy4wMDAwMDApIj4gICAgICAgIDxnIHRyYW5zZm9ybT0idHJhbnNsYXRlKDIxNS4wMDAwMDAsIDExOS4wMDAwMDApIi8+ICAgICAgICA8cGF0aCBkPSJNNTU1LjQ3NzI3Niw0MjEuMzU1OTU2IEM1NTYuNDM3MTIsNDE5Ljk3OTM4MyA1NTcsNDE4LjMwNTQyNSA1NTcsNDE2LjUgQzU1Nyw0MTEuODA1NTc5IDU1My4xOTQ0MjEsNDA4IDU0OC41LDQwOCBDNTQzLjgwNTU3OSw0MDggNTQwLDQxMS44MDU1NzkgNTQwLDQxNi41IEM1NDAsNDIxLjE5NDQyMSA1NDMuODA1NTc5LDQyNSA1NDguNSw0MjUgQzU1MC4zMDUxNDgsNDI1IDU1MS45Nzg4NjgsNDI0LjQzNzI5MyA1NTMuMzU1MzIxLDQyMy40Nzc3MTkgTDU1My4zNTU4OTIsNDIzLjQ3NzIxMiBMNTU5LjY1OTQzMyw0MjkuNzgwNzUzIEM1NTkuNzc2MzA5LDQyOS44OTc2MjkgNTU5Ljk2MjIwNiw0MjkuOTAxMjI1IDU2MC4wODIyMTEsNDI5Ljc4MTIyIEw1NjEuNzgxMjIsNDI4LjA4MjIxMSBDNTYxLjg5NzgzOCw0MjcuOTY1NTkzIDU2MS44OTI0MTcsNDI3Ljc3MTA5NyA1NjEuNzgwNzUzLDQyNy42NTk0MzMgTDU1NS40NzcyNzYsNDIxLjM1NTk1NiBaIE01NDguNSw0MjMgQzU1Mi4wODk4NTEsNDIzIDU1NSw0MjAuMDg5ODUxIDU1NSw0MTYuNSBDNTU1LDQxMi45MTAxNDkgNTUyLjA4OTg1MSw0MTAgNTQ4LjUsNDEwIEM1NDQuOTEwMTQ5LDQxMCA1NDIsNDEyLjkxMDE0OSA1NDIsNDE2LjUgQzU0Miw0MjAuMDg5ODUxIDU0NC45MTAxNDksNDIzIDU0OC41LDQyMyBaIiBmaWxsPSIjMzMzIi8+ICAgICAgICA8L2c+ICAgIDwvZz48L3N2Zz4=) no-repeat 98% center;
+}
+
+ca-list[searchable="true"] .ca-list-filter {
+    display: block;
+}
+
+ca-list[searchable="true"][data-count="0"] .ca-list-filter,
+ca-list[searchable="true"][data-count="1"] .ca-list-filter {
+    /* Hide the list filter if searchable AND 0 or 1 rows in the table */
+    display: none;
+}
+
+ca-list input.ca-list-filter {
+    box-sizing: border-box;
+    width: 100%;
+    display: block;
+    padding: 6px 12px;
+    font-size: 14px;
+    line-height: 1.42857143;
+    color: #333333;
+    background-color: #fff;
+    border: 1px solid #ccc;
+    border-radius: 4px;
+    -webkit-box-shadow: inset 0 1px 1px rgba(0, 0, 0, .075);
+    box-shadow: inset 0 1px 1px rgba(0, 0, 0, .075);
+    -webkit-transition: border-color ease-in-out .15s, -webkit-box-shadow ease-in-out .15s;
+    -o-transition: border-color ease-in-out .15s, box-shadow ease-in-out .15s;
+    transition: border-color ease-in-out .15s, box-shadow ease-in-out .15s;
+    display: none;
+}
+
+ca-list table[data-count]:after {
+    content: attr(data-count) " items in total";
+    display: block;
+    line-height: 1.5em;
+    position: absolute;
+    right: 2px;
+    bottom: -22px;
+    font-size: 0.9em;
+    color: #888;
+}
+
+ca-list table[data-count="1"]:after {
+    content: attr(data-count) " item in total";
+}
```

Comment:
Wow.. use of a feature that's been available for 7 years (in IE9), but took the death of IE8 to be able to actually use the thing!